src/qml/QFieldLocalDataPickerScreen.qml

```diff
@@ -474,9 +491,12 @@ Page {
 
         text: qsTr("Push to QFieldCloud")
         onTriggered: {
-          QFieldCloudUtils.addPendingAttachments(projectInfo.cloudUserInformation.username, cloudProjectsModel.currentProjectId, [itemMenu.itemPath], cloudConnection, true);
-          platformUtilities.uploadPendingAttachments(cloudConnection);
-          displayToast(qsTr("‘%1’ is being uploaded to QFieldCloud").arg(FileUtils.fileName(itemMenu.itemPath)));
+          QFieldCloudUtils.addPendingAttachments(cloudConnection.userInformation.username, cloudProjectsModel.currentProjectId, [itemMenu.itemPath], cloudConnection, true);
```

Comment:
OK, you'll need to change the cloudProjectsModel.currentProjectId to something like:

QFieldCloudUtils.getProjectId(projectActions.projectLocalPath)

And insure that the getProjectId() function there supports children path to a parent cloud project.